pkg/imager/widgets/dialogs.go

```diff
@@ -159,10 +167,17 @@ func ShowCustom(title, dismiss string, content fyne.CanvasObject, parent fyne.Wi
 		},
 	}
 	d.setButtons(widget.NewHBox(layout.NewSpacer(), d.dismiss, layout.NewSpacer()))
-
 	d.Show()
 }
 
+// ShowError displays an error dialog with a single OK button, that displays given text
+// The text is word wrapped
+func ShowError(text string, parent fyne.Window) {
+	label := widget.NewLabelWithStyle(text, fyne.TextAlignLeading, fyne.TextStyle{})
+	label.Wrapping = fyne.TextWrapWord
+	ShowCustom("Error blyat", "Ok", label, parent)
```

Comment:
Not sure we should add easter eggs to the Skyimager. Otherwise this works as expected and is cleaner than the previous PR. Good job.

Reply:
Thanks, that indeed is not a correct spelling. Fixed.